### PR TITLE
change actions to trigger workflows on this branch

### PR DESCRIPTION
**Pull request by eteq**
I *think* what I should have done was make this change at the time I cut the new v4.3.x branch.  @pllim can you confirm that was the intent? (or at least that this makes sense?)

### CODE REVIEW

File: .github/workflows/ci_workflows.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-    - main
+    - v4.3.x
     tags:
     - '*'
   pull_request:
```
Comment (pllim) [inline .github/workflows/ci_workflows.yml:9]: Yes, also this:

```suggestion
  pull_request:
  - v4.3.x
```

To be consistent with:

https://github.com/astropy/astropy/blob/5bce144c6c3e8697496eeab87e689989f1dc5549/.github/workflows/ci_workflows.yml#L3-L9

**Comment by eteq** [comment]
@pllim , can you advise if I am confused about how to use `skip-basebranch-check`? Or if I should be updating... something about that for the new branch too?

**Comment by pllim** [comment]
The `skip-basebranch-check` check logic only updates when you do a new push, which you just did, so it should pass now.

I had a reason to not have `labeled` trigger it but I don't quite remember now. Gotta dig at my original PR to find out if you must know.

**Comment by pllim** [comment]
p.s. It should skip base branch check automatically if the bot does the backport. 😉 

https://github.com/astropy/astropy/blob/9819e926cb49fbc36019f5e641b563975e56d57c/.github/workflows/ci_workflows.yml#L28-L31

**Review by pllim — APPROVED** [review]

**Comment by eteq** [comment]
The failure is real, but not related to this PR.  So I'm going to go ahead and merge just to make sure this triggers the CI as expected.

**Pull request merged**